### PR TITLE
create_vector_store_file_batch_and_poll + samples and update aio using the latest code from syn

### PR DESCRIPTION
**Pull request by howieleung**
create_vector_store_file_batch_and_poll + samples and update aio using the latest code from syn

### CODE REVIEW

File: sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_vector_store_batch_file_search_async.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        # if you do not specify a vector store, create_message will create a vector store with a default expiration policy of seven days after they were last active 
+        vector_store = await ai_client.agents.create_vector_store_and_poll(file_ids=[], name="sample_vector_store")
+        print(f"Created vector store, vector store ID: {vector_store.id}")
+        
+        # add the file to the vector store
+        vector_store_file_batch = await ai_client.agents.create_vector_store_file_batch_and_poll(vector_store_id=vector_store.id, file_ids=[file.id])    
+        print(f"Created vector store file batch, vector store file batch ID: {vector_store_file_batch.id}")
+        
+        # create a tool resources object with the file search tool
+        file_search_tool_resource = FileSearchToolResource(vector_store_ids=[vector_store.id])
```
Comment (jhakulin) [inline sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_vector_store_batch_file_search_async.py:69]: not sure if in this sample we want to show FileSearchTool + ToolSet

Comment (jhakulin) [reply sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_vector_store_batch_file_search_async.py:69]: maybe not as there is no many tools

**Review by jhakulin — APPROVED** [review]

File: sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_vector_store_batch_file_search_async.py (hunk 1 above)
Comment (jhakulin) [reply sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_vector_store_batch_file_search_async.py:69]:
```
    file_search_tool = FileSearchTool()
    file_search_tool.add_vector_store_list([vector_store.id])

    agent = ai_client.agents.create_agent(
        model="gpt-4-1106-preview", name="my-assistant", 
        instructions="You are helpful assistant",
        tools=file_search_tool.definitions,
        tool_resources=file_search_tool.resources
    )
```

Comment (howieleung) [reply sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_vector_store_batch_file_search_async.py]: Thanks Jarno..   I am making

```
file_search_tool = FileSearchTool(vector_store_ids=[vector_store.id])
```

**Pull request merged**